apps/calendar/js/main.js

```diff
@@ -1,25 +1,11 @@
-(function() {
-'use strict';
-
-function waitForLoad() {
-  return new Promise(accept => {
-    if (document.readyState === 'complete') {
-      return accept();
-    }
-
-    window.addEventListener('load', function onLoad() {
-      window.removeEventListener('load', onLoad);
-      return accept();
-    });
-  });
-}
-
-console.log('Will configure rjs...');
+window.require = window.require || window.curl;
```

Comment:
my idea here was that I was testing with both curl and alameda.. doing like this the code runs with any of the libs (unit tests still uses alameda..) better to keep it like this.